resources/views/hardware/view.blade.php

```diff
@@ -1039,7 +1039,7 @@ class="table table-striped snipe-table"
                       data-cookie-id-table="assetFileHistory">
                 <thead>
                   <tr>
-                    <th data-visible="true" data-field="icon"><span class="sr-only">Icon</span></th>
+                    <th data-visible="true" data-field="icon">{{trans('general.file_type')}}</th>
```

Comment:
SUPER tiny style detail - can you add spaces between the bracket and the string? So it's like:

```
{{ trans('general.file_type') }}
```

vs 

```
{{trans('general.file_type')}}
```